components/layout/AppMobileNavbar.vue

```diff
@@ -118,6 +121,9 @@ export default {
       this.$store.commit('alert/clear');
       context.toggleOpen();
     },
+    to(route) {
+      sameRouteReload(this, route, 'in');
```

Comment:
nitpick(blocking): 'in' is the default value for this param so you should leave it out here. Or, you could make this param null, i guess this function might be used in some other places as well.

Reply:
Good point. I completely forgot that I set this as a default.